CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+This document outlines major changes between releases.
```

Comment:
0.1.0 should be mentioned as initial one with a link.

Reply:
> 0.1.0 should be mentioned as initial one
Fixed.

> with a link.
Where to put the link?

Reply:
At the bottom, like
```
[Unreleased]: https://github.com/nspcc-dev/neofs-node/compare/v0.40.0...master
[0.40.0]: https://github.com/nspcc-dev/neofs-node/compare/v0.39.2...v0.40.0
[0.39.2]: https://github.com/nspcc-dev/neofs-node/compare/v0.39.1...v0.39.2
```

Reply:
Fixed.